app/src/renderer/texture.rs

```diff
@@ -0,0 +1,42 @@
+use core::ptr::null;
+
+use crate::renderer::{
```

Comment:
Wait, what is happening here? Is the renderer module re-exporting methods from Grrlib?

It seems better (less brittle, easier to understand) to import them in this module directly from its source.

Reply:
Fixed this the proper way by finally moving grrlib to a separate dependency named `grrustlib`, see the changes :)